src/state/queries/notifications/unread.tsx

```diff
@@ -70,12 +70,10 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
       },
 
       async checkUnread() {
-        const agent = getAgent()
-
-        if (!agent.session) return
+        if (!getAgent().session) return
```

Comment:
Seems like a code smell now, right? Ideally we render this unnecessary 🤔 maybe it's already not needed?